frontend/app/controllers/spree/checkout_controller.rb

```diff
@@ -159,9 +157,7 @@ def before_payment
         end
       end
 
-      if try_spree_current_user && try_spree_current_user.respond_to?(:payment_sources)
-        @payment_sources = try_spree_current_user.payment_sources
-      end
+      @payment_sources = try_spree_current_user.payment_sources if try_spree_current_user && try_spree_current_user.respond_to?(:payment_sources)
```

Comment:
you don't need `try_spree_current_user &&` here. `try_spree_current_user.respond_to?(:payment_sources)` will do the job since `nil` have method `respond_to?` so you can call it like this

Reply:
Okay I changed it.